overlord/ifacestate/handlers.go

```diff
@@ -107,7 +106,7 @@ func (m *InterfaceManager) doSetupProfiles(task *state.Task, _ *tomb.Tomb) error
 	if err := m.reloadConnections(snapName); err != nil {
 		return err
 	}
-	if err := m.autoConnect(task, snapName, blacklist); err != nil {
+	if err := m.autoConnect(task, snapName, nil); err != nil {
```

Comment:
added a FIXME here (also created a card in trello), didn't put two because unclear we will keep the structure when we fix this